final_prep/annotations.rb

```diff
@@ -4,40 +4,75 @@
 
 # Build a Bear
 
+# Creating a function called build_a_bear with 5 Arguments called name, age, fur,
+# clothes, and special_power.
```

Comment:
These are called _parameters_. Arguments are the values that you actually pass into the method (i.e. on lines 38 and 40). Parameters and arguments are 2 sides of the same coin, so to speak.